code/style.css

```diff
@@ -1,58 +1,214 @@
 body {
-  margin: 0;
+  margin: 0 auto;
   font-family: "Courier New", Courier, monospace;
+  background-color: #2e263a;
+  color:black;
+}
+
+header {
+  background-color: rgb(238, 228, 186);
+  text-align: center;
+  margin-bottom: 50px;
+}
+
+header h1 {
+  padding: 20px;
+  font-size: 36px;
 }
 
 p {
-  margin: 1px;
   font-weight: 300;
-  background-color: #fff;
+  background-color: grey;
+  font-size: 16px;
+  border-radius: 4px;
+  margin: 20px;  
 }
 
 .cards {
-  margin: 50px auto;
+  margin: auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
   display: flex;
   justify-content: space-around;
   align-items: center;
 }
 
+.allCards {
+  outline: none;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 2px 4px 6px grey;
+  margin-bottom: 10px;
+}
+
+.allCards:hover {
+  background-color: #3e8e41;
+}
+
+.allCards:active {
+  background-color: #3e8e41;
+  box-shadow: 3px 4px 5px #666;
+  transform: translateY(4px);
+}
+
 .killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+  width: 30%;
+  height: 530px;
+  background: rgb(212, 9, 9);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 25px;
   border-radius: 16px;
 }
 
 .weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
+  width: 30%;
+  height: 520px;
+  background: rgb(56, 175, 211);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 25px;
   border-radius: 16px;
 }
 
 .room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
-
+  width: 30%;
+  height: 520px;
+  background: rgb(68, 68, 216);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 25px;
   border-radius: 16px;
 }
 
+img.questionmark  {
+  filter:sepia(100%);
+}
+
+/*BOUNCY LOADER*/
+
+.wrapper {
+  width: 100%;
+  text-align: center;
+  margin-top: 30px;
+}
+
+.bouncing-ball {
+  position: relative;
+  width: 4rem;
+  margin: 0 auto;
+  height: 12rem;
+}
+
+.ball {
+  width: 4rem;
+  height: 4rem;
+  border-radius: 50%;
+  background: black;
+  position: absolute;
+  top: 0;
+  animation: bounce 1s infinite;
+}
+
+@keyframes bounce {
```

Comment:
This animation is a very great job Anna!

Reply:
Very much so agree!